
--------------------begin------------------------

[2020/01/03 09:27:57.88435][i][4c0a16fd8] recv loop terminated: err=failed to read from connection: read tcp 192.168.5.62:58098->192.168.0.101:2181: i/o timeout
[2020/01/03 09:27:57.88442][i][4c0a16fd8] send loop terminated: err=<nil>
[2020/01/03 09:27:57.88626][w][4c0a16fd8] zk已断开连接:[192.168.0.101]
[2020/01/03 09:27:59.88770][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:01.89113][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:03.89514][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:05.90067][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:07.90249][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:09.90541][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:11.90943][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:13.91242][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:15.91626][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:17.91982][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:19.92630][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:21.93047][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:23.93561][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:25.94376][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:27.95111][i][4c0a16fd8] Failed to connect to 192.168.0.101:2181: dial tcp 192.168.0.101:2181: i/o timeout
[2020/01/03 09:28:28.592535][i][4c0a16fd8] Connected to 192.168.0.101:2181
[2020/01/03 09:28:32.135461][i][4c0a16fd8] authentication failed: zk: session has been expired by the server
[2020/01/03 09:28:32.135483][w][4c0a16fd8] zk已断开连接:[192.168.0.101]
[2020/01/03 09:28:33.135863][i][4c0a16fd8] Connected to 192.168.0.101:2181
[2020/01/03 09:28:33.150448][i][4c0a16fd8] authenticated: id=103116747297988059, timeout=4000
[2020/01/03 09:28:33.150456][i][4c0a16fd8] re-submitting `0` credentials after reconnect
[2020/01/03 09:28:34.142137][i][f106e2da9] 开始启动[API]服务...
[2020/01/03 09:28:34.142265][e][f106e2da9] 启动失败 创建db失败:{
  "provider": "mysql",
  "connString": "oms_t:123456@tcp(192.168.0.36)/oms_t?charset=utf8",
  "maxOpen": 20,
  "maxIdle": 10,
  "lifeTime": 600
},err:sql: unknown driver "mysql" (forgotten import?)

---------------------end-------------------------
